### PR TITLE
#44 Add shopping list an estimated price for hardware components

### PR DESCRIPTION
**Pull request by blckfox**
# Description

See #44

### CODE REVIEW

**Comment by blckfox** [comment]
@xKabbe could you review the shopping list? Please do not merge yet. It may requires more work. If you think the price is good I can buy the parts. There is a "Laboratory Power Supply" at the bottom of the list we should talk about. Pls contact me about this.

**Comment by blckfox** [comment]
Issue #44 requires the products to be shipped and arrived to be closed.

File: Hardware/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Hardware related general Documentation
+
+### Shopping List
+
+
```
Comment (xKabbe) [inline Hardware/README.md:5]: It seems that you did not save you changes before commiting it :)

**Review by xKabbe — COMMENTED** [review]
@blckfox Review is finished

**Comment by blckfox** [comment]
@xKabbe please review and merge. Please reopen closed related issue #44 after merge.

**Review by xKabbe — APPROVED** [review]

**Pull request merged**